single-node/kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+clusters:
+- cluster:
+    certificate-authority: ssl/ca.pem
+    server: https://172.17.4.99:443
+  name: vagrant-cluster
+contexts:
+- context:
+    cluster: vagrant-cluster
```

Comment:
vagrant, or vagrant-single, or vagrant-cluster, or vagrant-single-cluster. Don't feel strongly but keep it consistent between all examples.